### DISCUSSION TITLE
Update-2022-10-12

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#130] [Announcements]
I implemented `isosurface` mesh class (aka blobbies, metaballs). Demos in `demo-isosurface.lisp`.
![Screen Shot 2022-10-12 at 2 19 20 PM](https://user-images.githubusercontent.com/103860793/195477370-aa6019ee-ce47-43be-a375-3e1a357e9235.png)

**Comment by aykaramba** [comment]
Wow.

**Comment by kaveh808** [comment]
Using curve-source-protocol to generate isosurface.

![Screen Shot 2022-10-12 at 4 46 00 PM](https://user-images.githubusercontent.com/103860793/195703221-57411709-fba9-4fa8-b384-3df8343d3f81.png)

**Comment by kaveh808** [comment]
Using particle system as curve-source-protocol.

![Screen Shot 2022-10-13 at 10 08 30 AM](https://user-images.githubusercontent.com/103860793/195703414-f67de5fd-8fdf-45e9-8a9e-ea695faaa635.png)
